importer/wait-for

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
```

Comment:
This was suggested to wait for the db to become available which allows us to do the import _after_ the DB as started as part of starting up all the services. The app will fail continuously until the initial data ingestion is complete, but thanks to pm2 it keeps retrying until it can connect.